Taskfile.yml

```
@@ -81,18 +83,11 @@ tasks:
-        git clone
-        https://github.com/emscripten-core/emsdk.git
-        --branch 3.1.67
-        --depth 1
-        --shallow-submodules
-        "{{.G_EMSDK_DIR}}"
+      - "git clone https://github.com/emscripten-core/emsdk.git '{{.G_EMSDK_DIR}}'"
       - |-
         cd "{{.G_EMSDK_DIR}}"
-        ./emsdk install latest
-        ./emsdk activate latest
+        ./emsdk install {{.EMSDK_VERSION}} 
```

yamllint

[error] trailing spaces

(trailing-spaces)